messages/handler/on_message.go

```diff
@@ -21,12 +21,8 @@ import (
 	"errors"
 	"fmt"
 
```

Comment:
Maybe it is better to delete the whole file and move the rest to the functions to other modules:
- `InterfaceArrayToStringArray` could be moved to a utils module (`misc.go`?) as it could be used by any module
- `InterfaceToPlugins` is a useless artifact that can be removed
- `ParseMessageType` a bit more complicated as there are 2 different methods with this name. My suggestion would be to rename one of them and put both in the `messages.go` module